mimic/rest/identity_api.py

```diff
@@ -250,6 +250,43 @@ def get_users_details(self, request):
             "updated": seconds_to_timestamp(time.time())
         }))
 
+    @app.route('/v2.0/users/<string:user_id>/OS-KSADM/credentials',
+               methods=['GET'])
+    def get_user_credentials_osksadm(self, request, user_id):
+        """
+        Support, such as it is, for the credentials call.
+
+        reference: http://developer.openstack.org/api-ref-identity-v2-ext.html
+            #listCredentials
+        """
+        if user_id in self.core.sessions._userid_to_session:
+            username = self.core.sessions._userid_to_session[user_id].username
+            password = '9d5ed678fe57bcca610140957afab571'  # echo -n B | md5sum
+            apikey = '0d61f8370cad1d412f80b84d143e1257'  # echo -n C | md5sum
+            return json.dumps(
+                {
+                    'credentials': [
+                        {
+                            'passwordCredentials': {
```

Comment:
When I saw this part - `passwordCredentials` - I did a double-take.  ''Surely'' you must be mistaken and OpenStack would not actually have an extension for saving your plaintext credentials and relaying them back to you.  Alas, it seems that is accurate.  But at least, it seems the Rackspace cloud won't give them back.  Do you actually need `passwordCredentials` in here, or were you just leaving them there because the API docs said they'd be there?  Actually returning this value would require a badly misconfigured authentication backend that would write your credentials out in plain text, so I think we should default to not returning it.